types/supertest/tslint.json

```diff
@@ -4,6 +4,7 @@
         "ban-types": false,
         "dt-header": false,
         "no-empty-interface": false,
+        "no-outside-dependencies": false,
```

Comment:
Not sure why this is needed, the supertest types don't import from `buffer`. The only direction I could find in resolving it was https://github.com/DefinitelyTyped/DefinitelyTyped/issues/49372.